openapi/2023-12.yaml

```
@@ -570,6 +576,71 @@ paths:
           $ref: '#/components/responses/HttpError'
         5XX:
           $ref: '#/components/responses/HttpError'
+  '/lists/{id}/phone-numbers':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: Id of the list in Recart
+    post:
+      description: Handle phonenumber operations in Recart lists
+      summary: Handle phoneNumber operations in Recart lists
+      tags:
+        - List
+      responses:
+        '200':
+          description: ''
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/PhoneNumbersToListResponse'
+              examples:
+                Example 1:
+                  value:
+                    type: lists
+                    attributes:
+                      results:
+                        - phoneNumber: '+1311111111'
+                          status: added
+                        - phoneNumber: '+1311111111'
+                          status: removed
+                        - phoneNumber: '+1311111111'
+                          status: not_found
+        4XX:
+          $ref: '#/components/responses/HttpError'
+        5XX:
+          $ref: '#/components/responses/HttpError'
+      operationId: postList
```

postList is for list creation
```suggestion
      operationId: modifyListMembers
```